assets/multiqc_config.yml

```
@@ -1,18 +1,18 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/quantms/releases/tag/1.2.0" target="_blank">nf-core/quantms</a>
+  This report has been generated by the <a href="https://github.com/nf-core/quantms/tree/dev" target="_blank">nf-core/quantms</a>
```

```suggestion
  This report has been generated by the <a href="https://github.com/bigbio/quantms/releases/tag/1.3.0" target="_blank">nf-core/quantms</a>
```
Same thing down there.